DNN Platform/DotNetNuke.Abstractions/Prompt/ICommandOption.cs

```diff
@@ -0,0 +1,29 @@
+﻿// 
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// 
+namespace DotNetNuke.Abstractions.Prompt
+{
+    /// <summary>
+    /// This is used in the ICommandHelp to send a list of command parameters to the client for explanatory help
+    /// </summary>
+    public interface ICommandOption
```

Comment:
Not sure of the naming here.  This is named ICommandOption, but everywhere inside this object it is referred to itself as a Parameter?